firefox_ui_harness/decorators.py

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import SkipTest
+
+from firefox_puppeteer.api import appinfo
+
+
+def skip_if_e10s(target):
+    def wrapper(self, *args, **kwargs):
+        if appinfo.AppInfo.browserTabsRemoteAutostart:
```

Comment:
There's no instance, this is broken right now. I don't know that we're going to need that many properties of appinfo that aren't supplied by session_capabilities, I sort of wonder this is worth it right now vs. the original version.